app/assets/stylesheets/frontend/layouts/html-publication.scss

```diff
@@ -25,3 +25,9 @@
     padding-top: 0;
   }
 }
+
+#print-meta-data {
```

Comment:
A heads up, these styles don't pass our sass lint checks (which are in government-frontend)

Sass linting prevents:
* styling based on ID
* adding units to `0`s

Why did you opt for `visibility: hidden;` over `display: none;`?